### ISSUE TITLE
Add Twitter Card support

### ISSUE THREAD

**Issue by chayev** [chayev/yurl#2]
Using a website reference, check if that website has proper Twitter Card support. This would allow you to confirm that your site will display properly when shared on Twitter.

Use case: `yurl twittercard validate <location>`

Alias: tc 

Output:

- Valid - return whether the Twitter Card data format is valid (based on [Reference](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary))
- Invalid - return whether the Twitter Card data format is invalid based on above reference
- Not Found - tags not found

**Comment by FelicianoTech** [comment]
@chayev You want me to take this one?

**Comment by chayev** [comment]
@felicianotech yes. Let me know if there are any questions as you work through it.
